src/data/faq_de.json

```diff
@@ -292,8 +298,13 @@
 			"Siehe auch:",
 			"<ul><li><a href='https://www.coronawarn.app/de/faq/#no_risk_update_ios'>Apple: Mein Risikostatus wurde seit über einem Tag nicht mehr aktualisiert. Internet war aber verfügbar, was läuft hier falsch?</a></li><li><a href='https://www.coronawarn.app/de/faq/#no_risk_update'>Android: Mein Risikostatus wurde seit über einem Tag nicht mehr aktualisiert. Internet war aber verfügbar, was läuft hier falsch?</a></li></ul>"
                     ]
-                },
-                {
+                },{
+                    "title": "Was passiert, wenn ich Bluetooth (und bei Android den Standort) ausschalte?",
+                    "anchor": "bluetooth_off",
+                    "textblock": [
+                      "Begegnungen mit anderen Smartphones, die die Corona-Warn-App laufen haben, können nicht aufgezeichnet werden, weil keine Zufallscodes ausgetauscht werden. Haben Sie deswegen bitte immer Bluetooth (und bei Android den Standort) aktiviert, wenn Sie Kontakt zu anderen Leuten haben, zum Beispiel, wenn Sie Gäste bekommen oder Sie unterwegs sind."
```

Comment:
"Haben Sie deswegen bitte immer Bluetooth (und bei Android den Standort) aktiviert, wenn Sie Kontakt zu anderen Leuten haben, zum Beispiel, wenn Sie Gäste bekommen oder Sie unterwegs sind."
->
"Bitte lass deswegen immer Bluetooth (und bei Android den Standort) aktiviert, wenn Du Kontakt zu anderen Leuten hast, zum Beispiel, wenn Du Gäste empfängst oder unterwegs bist."

(we use the informal "Du" in the FAQs)